models/yang/annotations/sonic-neigh-annot.yang

```diff
@@ -0,0 +1,16 @@
+module sonic-neigh-annot {
+
+    yang-version "1";
+
+    namespace "http://openconfig.net/yang/annotation/sneigh-annot";
+    prefix "sneigh-annot";
+
+    import sonic-neigh { prefix sneigh; }
+    import sonic-extensions { prefix sonic-ext; }
+
+    deviation /sneigh:sonic-neigh/sneigh:NEIGH_TABLE {
+        deviate add {
+            sonic-ext:db-name "APPL_DB";
```

Comment:
Annotation file should be generally used for standard YANG model or for some transformer specific extension. This extension should directly go into the SONIC YANG.

@kwangsuk  What is your opinion ?

Reply:
These changes were in sonic-yang, but @kwangsuk  and @joyas-joseph  suggested to create an annotation file for that.

Reply:
@dutta-partha The extension sonic-ext:db-name is used by Transformer to locate the target DB. We would keep all those Get/Set transformer hints in the annotation file. We also plan to support more Transformer extensions to the SONiC yang down the road as needed.